sdk/synapse/synapse-access-control/src/accessControlClientContext.ts

```diff
@@ -4,7 +4,7 @@ import * as coreHttp from "@azure/core-http";
 import { AccessControlClientOptionalParams } from "./models";
 
 const packageName = "@azure/synapse-access-control";
-const packageVersion = "1.0.0-beta.1";
+const packageVersion = "1.0.0-beta.2";
```

Comment:
This has a breaking change. But, I undetstand it is a beta version. So, if it is Ok to introduce breaking change (in beta) and incrementing the version to .2 from .1, then I am fine. Please confirm

Reply:
The upcoming release will be beta 2, I'm just making sure all the versions align so if we see it in the logs we'll know which one we're looking at.